README.md

```diff
@@ -29,26 +29,121 @@
+#### Creates the development and test database
+`$ make create-db`
 
-## Como Contribuir
+#### Drops down the docker environment
+`$ make down`
 
-Para contribuir com o Falko, o colaborador deve criar seu próprio Fork e enviar um [pull request](https://github.com/fga-gpp-mds/Falko-2017.2-BackEnd/pulls) com a contribuição para a branch 'devel' do projeto.
-O código enviado será submetido a uma avaliação por um dos administradores do repositório e, se aprovado, incluído ao _core_ da aplicação.
+#### Migrates the Rails database
+`$ make migrate`
 
-## Licença
+#### Lists all your running services
+`$ make ps`
+
+#### Removes all your docker networks
+`$ make rm-network`
+
+#### Removes all your docker mapped volumes
+`$ make rm-volume`
+
+#### Creates the Falko environment and run it
+`$ make run`
+
+#### Creates the Falko environment and run it as daemon
+`$ make quiet-run`
+
+#### Creates only the API environment, without database service
+`$ make run-api`
+
+#### Creates only the Postgres database environment, without API service
+`$ make run-db`
+
+#### Seeds the Rails database environment
+`$ make seed`
+
+#### Executes all of Rails' tests
+`$ make test`
+
+### Docker Usage
+* Download and install Docker CE at the [official site](https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/#install-from-a-package).
+
+* Download and install Docker Compose at the [official site](https://docs.docker.com/compose/install/#master-builds).
+
+#### Clone the repository and enter it
+```
+git clone https://github.com/falko-org/Falko-API && cd Falko-API/
+```
+
+#### Make sure you're at _devel_ branch
+```
+git checkout devel
+```
+
+#### Lift your environment
+```
+docker-compose up
+```
+
+#### Useful commands
+#### How to download a docker image
+```
+docker pull imageYouWant
+```
+
+#### Listing local images
+```
+docker images
+```
+
+#### Deleting images
+```
+docker rmi -f imageId
+```
+
+#### Listing running containers
+```
+docker ps
+```
+
+#### Removing containers
+```
+docker rm [-f] containerNameOrId
+```
+
+#### Executing commands from outside the container
+```
+docker exec <container-name> <desired-command>
+```
+Example:
+```
+docker exec falko-api rails generate model User name:string
+```
+
+## Documentation
+
+Additional documentation is avaiable at [Official Wiki](https://github.com/fga-gpp-mds/Falko-2017.2-BackEnd/wiki).
+
+## How to Contribute
+
+To contribute with us, the colaborator must _fork_ and send a [pull request](https://github.com/fga-gpp-mds/Falko-2017.2-BackEnd/pulls) with his/her contribution to _devel_ branch.
+The code will be analized by one of the project's owners and, if approved, included to the application's core.
+
+## License
 
 [MIT](https://github.com/fga-gpp-mds/Falko-2017.2-BackEnd/blob/devel/LICENSE)
 
-Copyright (c) 2017 Falko - UnB-FGA (GPP/MDS) Grupo 8 Segundo Semestre de 2017
+Copyright (c) 2018 Falko Organization
```

Comment:
Here, you have placed 2018 in the year of the copyright and the LICENSE is with the year 2017, @alaxalves.